framework/src/mesh/MooseMesh.C

```diff
@@ -140,6 +140,14 @@ validParams<MooseMesh>()
                         " when DistributedMesh is used. Value is ignored in ReplicatedMesh mode");
   params.addParam<unsigned int>(
       "patch_size", 40, "The number of nodes to consider in the NearestNode neighborhood.");
+  params.addParam<bool>("automatic_patch_update",
+                        false,
+                        "Updates the patch for slave nodes for which penetration is not detected.");
+  params.addParam<unsigned int>("max_leaf_size",
+                                10,
+                                "The leaf size determines the maximum number "
+                                "of points in each leaf of the KDTree used in "
+                                "the nearest neighbor search.");
```

Comment:
Strike 'The leaf size determines the'. Also, maybe put a very short explanation on how changing that number would affect performance.